cypress/integration/plugins/security-dashboard-plugin/aggregation_view.js

```diff
@@ -0,0 +1,352 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  INDEX_PATTERN_PATH,
+  SAVED_OBJECTS_PATH,
+  SEC_UI_TENANTS_PATH,
+  SEC_UI_USER_CREATE_PATH,
+  SEC_UI_ROLES_CREATE_PATH,
+  SEC_UI_ROLE_EDIT_PATH,
+} from '../../../utils/constants';
+
+import { 
+  ADMIN_AUTH,
+  CURRENT_TENANT,
+ } from '../../../utils/commands';
+
+const tenantName = 'test';
+const tenantDescription = 'Test description';
+const testIndexPattern1 = 's';
+const testIndexPattern2 = 'se';
+const userName1 = 'test1';
+const userName2 = 'test2';
+const password = 'password';
+const roleName1 = 'roleWithTest';
+const roleName2 = 'roleWithoutTest';
+const clusterPermission = 'cluster_all';
+const indexName = '*';
+const indexPermission = 'indices_all';
+
+if (Cypress.env('SECURITY_ENABLED') && Cypress.env('AGGREGATION_VIEW')) {
+  describe('Saved objects table test', () => {
+    // start a server so that server responses can be mocked via fixtures
+    // in all of the below test cases
+    before(() => {
+      cy.server();
+    });
+
+    it('should create new tenant successfully by selecting `Create tenant`', () => {
+      cy.visit(SEC_UI_TENANTS_PATH);
+  
+      cy.get('button[id="createTenant"]').first().click({ force: true });
+  
+      cy.contains('button', 'Create');
+      cy.contains('button', 'Cancel');
+      cy.contains('.euiModalHeader__title', 'Create tenant');
+  
+      cy.get('input[data-test-subj="name-text"]').type(tenantName, {
+        force: true,
+      });
+      cy.get('input[data-test-subj="name-text"]').should(
+        'have.value',
+        tenantName
+      );
+  
+      cy.get('textarea[data-test-subj="tenant-description"]').type(
+        tenantDescription,
+        { force: true }
+      );
+      cy.get('textarea[data-test-subj="tenant-description"]').should(
+        'have.value',
+        tenantDescription
+      );
+  
+      cy.get('button[id="submit"]').first().click({ force: true });
+  
+      cy.url().should((url) => {
+        expect(url).to.contain('/tenants');
+      });
+  
+      cy.contains('h3', 'Tenants');
+      // should contain the new tenant that was just created
+      cy.contains('.euiTableCellContent', tenantName);
+      cy.contains('span', tenantDescription);
+    });
+
+    it('should create new user called test1 successfully', () => {
```

Comment:
this test seems to be exact the same as the below `should create new user called test2 successfully` test except for using different user name, could you please just keep one test?